index.html

```diff
@@ -178,36 +178,36 @@ <h1>Uber Navigation</h1>
                     <div class="connect">LET'S CONNECT</div>
```

Comment:
- [ ] Kindly as the previous reviewer asked, you have low contrast at this div `<div class="connect">LET'S CONNECT</div>` so you just need to change the color of the item to a darker one to pass. you can use `#865AE5` for the div text.

![image](https://user-images.githubusercontent.com/35535130/156378940-91c2e875-245b-4f7e-9c45-a8a0d423e92d.png)